### PR TITLE
DOD code merge to run 21.

### PR DESCRIPTION
**Pull request by lg345**

### CODE REVIEW

File: mfx/beamline.py [hunk 1]
```diff
@@ -74,6 +74,25 @@
...
+
+    # Hack the LXE class to make it work with Newports
+    class LXE(LaserEnergyPositioner):
+        motor = Cpt(Newport, "")
+
+    lxe_calib_file = (
+        f"/reg/neh/operator/mfxopr/experiments/{get_current_experiment('mfx')}/wpcalib"
+    )
+    try:
+        lxe_pw = LXE("MFX:LAS:MMN:08", calibration_file=lxe_calib_file, name="lxe_pw")
```
Comment (gadorlhiac) [inline mfx/beamline.py:91]: Are people happy with this nomenclature? When I added it in I didn't want to conflict with any objects others may have been trying to create at the time. If you'd prefer `lxe` to `lxe_pw` we should probably make that change before merging. It's probably useful to keep things consistent with muscle memory people have from moving between hutches?

File: mfx/beamline.py~
Comment (gadorlhiac) [inline mfx/beamline.py~:1]: This file should be untracked and removed. It's a backup emacs file which is not needed in the repo.

File: mfx/focus_scan.py
Comment (gadorlhiac) [inline mfx/focus_scan.py:1]: Looks fine to me. This was tested and found to still work during `mfxx1006522` right?

**Review by gadorlhiac — CHANGES_REQUESTED** [review]
Looks fine to me - @lg345 can you decide on the naming for the `lxe` object? Otherwise a couple quick changes and it can be merged.

File: mfx/beamline.py (hunk 1 above)
Comment (lg345) [reply mfx/beamline.py:91]: lxe will be the final name for the object.

**Pull request merged**